src/pages/HomePage/Home.tsx

```diff
@@ -2,27 +2,24 @@ import "./home.scss";
 import React, { useState, useEffect } from "react";
 import "bootstrap/dist/css/bootstrap.min.css";
 import SliderComponent from "../../components/ui/slider/Slider";
+import Loader from "../../components/ui/loader/Loader";
 import { BreedType } from "../../types";
 
 function Home() {
-  const [breed, setInfo] = useState<BreedType[]>([]);
-  const [errorMessage, setError] = useState<string>("");
+  const [breed, setBreed] = useState<BreedType[]>([]);
   useEffect(() => {
     fetch("https://api.thedogapi.com/v1/breeds?limit=10&page=0")
       .then((res) => res.json())
-      .then((data) => setInfo(data))
+      .then((data) => setBreed(data))
       .catch((error) => {
         console.error("Error fetching breed information:", error);
-        setError(String(error.errorMessage));
       });
-  }, [errorMessage]);
+  }, [breed]);
 
   return (
-    <div className="home container-fluid">
+    <div className="home">
       <h1 className="home__title">home page</h1>
-      {errorMessage &&
-        "Oops, something went wrong. Error message: " + errorMessage}
-      <SliderComponent breed={breed} />
+      {breed.length ? <SliderComponent breed={breed} /> : <Loader />}
```

Comment:
since SliderComponent is UI component, property `breed` should be renamed to more generic name like `items` or `slides` to make it reusable for other cases